course/api/full_serializers.py

```diff
@@ -109,6 +110,19 @@ class Meta(CourseUsertagBriefSerializer.Meta):
             'taggings',
         )
 
+    def validate(self, data):
+        # Name is enough, slug will be automatically generated 
+        data = super().validate(data)
+        if not data['name']:
+            raise serializers.ValidationError(
+                'At minimum, field \'name\' is required'
```

Comment:
I think this can be on a single line. Any case, if you use `"` for string, then single quotes don't need to be escaped. It's good to follow [this "rule"](https://stackoverflow.com/a/56190/11090030) when choosing quotes in Python. It makes the code really easy to read :)